pkg/utils/k8sutil.go

```diff
@@ -210,6 +210,9 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 	labels := map[string]string{
 		"function": name,
 	}
+	for k, v := range funcObj.Metadata.Labels {
```

Comment:
Oh no I should do a sort, otherwise there is a duplicate of `function=` in the label. Ok updating it.